tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -0,0 +1,382 @@
+        batch_size=8,
+        nr_loader_worker=0,
+        model=None,
+        predefined_model=None,
+        pretrained_weight=None,
+        verbose=True,
+    ):
+        """Initialise the Patch Predictor. Note, if model is supplied in the
+        arguments, it will override the backbone.
+
+        Args:
+            model (nn.Module): use externally defined PyTorch model for prediction with.
+                weights already loaded. Default is `None`. If provided,
+                `pretrained_model` argument is ignored.
+
+            predefined_model (str): name of the existing models support by tiatoolbox
+                for processing the data. Currently supports:
+                - alexnet-kather100K: alexnet backbone trained on Kather 100K dataset.
+                - resnet18-kather100K: resnet18 backbone trained on Kather 100K dataset.
+                - resnet34-kather100K: resnet34 backbone trained on Kather 100K dataset.
+                - resnet50-kather100K: resnet50 backbone trained on Kather 100K dataset.
+                - resnet101-kather100K: resnet101 backbone trained on
+                    Kather 100K dataset.
+                - resnext5032x4d-kather100K: resnext50_32x4d backbone trained on
+                    Kather 100K dataset.
+                - resnext101_32x8d-kather100K: resnext101_32x8d backbone trained on
+                    Kather 100K dataset.
+                - wide_resnet50_2-kather100K: wide_resnet50_2 backbone trained on
+                    Kather 100K dataset.
+                - wide_resnet101_2-kather100K: wide_resnet101_2 backbone trained on
+                    Kather 100K dataset.
+                - densenet121-kather100K: resnet101 backbone trained on
+                    Kather 100K dataset.
+                - densenet161-kather100K: densenet161 backbone trained on Kather
+                    100K dataset.
+                - densenet169-kather100K: densenet169 backbone trained on
+                    Kather 100K dataset.
+                - densenet201-kather100K: densenet201 backbone trained on
+                    Kather 100K dataset.
+                - mobilenet_v2-kather100K: mobilenet_v2 backbone trained on
+                    Kather 100K dataset.
+                - mobilenet_v3_large-kather100K: mobilenet_v3_large backbone trained on
+                    Kather 100K dataset.
+                - mobilenet_v3_small-kather100K: mobilenet_v3_small backbone trained on
+                    Kather 100K dataset.
+                - googlenet-kather100K: googlenet backbone trained on
+                    Kather 100K dataset.
+
+                By default, the corresponding pretrained weights will also be
+                downloaded. However, you can override with your own set of weights
+                via the `pretrained_weight` argument. Argument is case insensitive.
+
+            pretrained_weight (str): path to the weight of the corresponding
+                `predefined_model`.
+
```

Comment:
This is not rendering correctly on sphinx/readthedocs. There must be some spacing or indentation issue.